libraries/networking/src/AssetResourceRequest.cpp

```diff
@@ -0,0 +1,47 @@
+//
+//  AssetResourceRequest.cpp
+//
+//  Created by Ryan Huffman on 2015/07/23
+//  Copyright 2015 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+#include "AssetResourceRequest.h"
+
+#include "AssetClient.h"
+#include "AssetRequest.h"
+
+void ATPResourceRequest::doSend() {
```

Comment:
match this to the filename, comment block and include guards (they can be ATPResourceRequest if you want they should just be the same)